qt5-apps/qt-creator/DEPENDS

```diff
@@ -1 +1,9 @@
-depends  qt5
+depends qt5
+depends llvm
```

Comment:
AFAIK and not heard any chatter from kde/qt folks, llvm is not a hard depends. qtcreator automatically detects gcc when launched and any other compilers you have installed require manually adding via Tools -> Options. Will these changes make llvm the default?